docs/src/content/docs/book/statements.mdx

```diff
@@ -96,6 +98,157 @@ value += 5;     // augmented assignment (one of the many, see below)
+fun basic() {
+    // Basic syntax of destructuring assignment (to the left of "="):
+    let Example { number } = get42();
+    //  -------   ------     -------
+    //  ↑         ↑          ↑
+    //  |         |          gives the Example Struct
+    //  |         definition of "number" variable, derived
+    //  |         from the field "number" on Example Struct
+    //  target structure type "Example"
+    //  to destructure fields from
+
+    // Same as above, but with an instantiation
+    // to showcase how destructuring syntax mirrors it:
+    let Example { number } = Example { number: 42 };
+    //                       ----------------------
+    //                       ↑
+    //                       instantiation of Example Struct
+
+    // Above examples of syntax are roughly equivalent
+    // to the following series of statements:
+    let example = Example { number: 42 };
+    let number = example.number;
+}
+```
+
+Just like in [instantiation](/book/expressions#instantiation), the trailing comma is allowed.
+
+```tact
+struct Example { number: Int }
+
+fun trailblazing() {
+    let Example {
+        number,     // trailing comma inside variable list
+    } = Example {
+        number: 42, // trailing comma inside field list
+    };
+}
+```
+
+:::note
+
+  [Augmented assignment operators](/book/operators#augmented-assignment) do not make sense for such assignments and will therefore be reported as parsing errors:
+
+  ```tact
+  struct Example { number: Int }
+  fun get42(): Example { return Example { number: 42 } }
+
+  fun basic() {
+      let Example { number } += get42();
+      //                     ^ this will result in the parse error:
+      //                     expected "="
+  }
+  ```
+
+:::
+
+To create a binding under a different variable name, specify it after the semicolon `:{:tact}`.
+
+```tact
+// Similar definition, but this time field is called "field", not "number"
+struct Example { field: Int }
+
+fun naming(s: Example) {
+    let Example { field: varFromField } = s;
+    //                   ------------     ↑
+    //                   ↑                |
+    //                   |                instance of Example Struct, received
+    //                   |                as a parameter of the function "naming"
+    //                   definition of "varFromField" variable, derived
+    //                   from the field "field" on Example Struct
+}
+```
+
+Note, that the order of bindings doesn't matter — all the fields retain their values and types under their names no matter the order in which they stand in their definition in the respective [Struct][s] or [Message][m].
+
+```tact
+// "first" goes first, then goes "second"
+struct Two { first: Int; second: String }
+
+fun order(s: Two) {
+    let Two { second, first } = s;
+    //        ------  -----
+    //        ↑       ↑
+    //        |       this variable will be of type Int,
+    //        |       same as the "first" field on Struct Two
```

Comment:
Which preposition is the correct one? 
"field on Struct Two"   or
"field in Struct Two"
=)